### ISSUE TITLE
[MU4 Issue] Accessing 'Properties' via F8 on keyboard needs two keystrokes

### ISSUE THREAD

**Issue by miwelte** [musescore/MuseScore#15439]
**Describe the bug**
A short and clear description of what the bug is.

**To Reproduce**
1. Open existing score
2. Ensure additionals 'Views' are activated  and displayed in the left view panel
3. Select any other tab than 'Properties' from the left view panel.
4. Select one or more elements or bars wirh elements within the score
5. Press <kbd>F8</kbd> once
6. Press <kbd>F8</kbd> one more time directly afterwards

**Expected behavior**
Panel switches to 'Properties' as active tab and the available functions/properties are displayed immediately (e.g. General, Appearance, Notation, ...) after pressing <kbd>F8</kbd> only once.

**Platform information**
 - OS: Windows 10 22H2, Dell 5520 Xeon E3-1505M 32GB RAM
 - MU4 revision: github-musescore-musescore-5485621

**Additional context**
After pressing the <kbd>F8</kbd> for the first time, the 'Properties' tab diassapears from the panel temporarily and comes back into view as activated wehe pressing <kbd>F8</kbd> the second directly afterwards.

**Comment by BerserkGoat8** [comment]
I believe that this is the intended behaviour as the keyboard shortcut is described as "show/hide properties", as opposed to switching to the tab.

**Comment by cbjeukendrup** [comment]
This problem/annoyance is already covered by #11218. Thanks for reporting though!

**Comment by miwelte** [comment]
> This problem/annoyance is already covered by #11218. Thanks for reporting though!
Ok, perfect. Could not find anything despite search. Sorry for the duplicate.
